File: comms/email/fetch-email/fetch-email.java
package com.example.usage;

import com.twilio.comms.TwilioCommsClient;

public class Example {
    public static void main(String[] args) {
        TwilioCommsClient client = TwilioCommsClient
            .builder()
            .credentials("<username>", "<password>")
            .build();

        client.emails().fetch("comms_email_01h9krwprkeee8fzqspvwy6nq8");
    }
}

File: comms/email/fetch-email/fetch-email.cs
using TwilioComms;
using System.Threading.Tasks;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioCommsClient(
            accountId: "<username>",
            authToken: "<password>"
        );

        await client.Emails.FetchAsync(
            "comms_email_01h9krwprkeee8fzqspvwy6nq8"
        );
    }

}
